src/robot/parsing/model/statements.py

```diff
@@ -47,7 +47,7 @@ class Node(ast.AST, ABC):
 
 
 class Statement(Node, ABC):
-    _fields = ('type', 'tokens')
+    _attributes = (*Node._attributes, 'tokens', 'type')
```

Comment:
We still need to double check this change.

Reply:
yep, the attribute field is not used in RF in the `ast` module it is only used for duming the model.